StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1019MemberAccessSymbolsMustBeSpacedCorrectly.cs

```diff
@@ -96,23 +97,9 @@ private void HandleQuestionToken(SyntaxTreeAnalysisContext context, SyntaxToken
 
         private void HandleMemberAccessSymbol(SyntaxTreeAnalysisContext context, SyntaxToken token)
         {
-            bool precededBySpace;
-            bool firstInLine;
-
-            bool followedBySpace;
-
-            firstInLine = token.HasLeadingTrivia || token.GetLocation()?.GetMappedLineSpan().StartLinePosition.Character == 0;
-            if (firstInLine)
-            {
-                precededBySpace = true;
-            }
-            else
-            {
-                SyntaxToken precedingToken = token.GetPreviousToken();
-                precededBySpace = precedingToken.HasTrailingTrivia;
-            }
-
-            followedBySpace = token.HasTrailingTrivia;
+            bool firstInLine = token.IsFirstInLine();
+            bool precededBySpace = firstInLine ? true : token.IsPrecededBySpace();
```

Comment:
You could also write this like `precededBySpace = firstInLine || token.IsPrecededBySpace();`